lib/p2p/packets/HelloPacket.ts

```diff
@@ -0,0 +1,30 @@
+import Packet from './Packet';
+import PacketType from './PacketType';
+
+type HelloPacketBody = {
+  version: string;
+  nodeKey: string;
+  listenPort: number;
+  pairs: string[];
+};
+
+class HelloPacket extends Packet {
+  public type: PacketType = PacketType.HELLO;
```

Comment:
For consistency I think it looks better to make the type inferred here, aka drop the `: PacketType` to match with `responseType` below.

Reply:
Because we've set on explicit properties typing, I added types instead of dropping then (temp solution, this area will probably be changed in the next PR).